projects/portal/src/app/models/cosmos/bank.application.service.ts

```diff
@@ -39,18 +39,17 @@ export class BankApplicationService {
       return;
     }
 
-    if (!validatePrivateStoredWallet(privateWallet)) {
-      this.snackBar.open('Invalid Wallet info!', 'Close');
-      return;
-    }
-
     const privateKey = convertHexStringToUint8Array(privateWallet.privateKey);
-
     if (!privateKey) {
       this.snackBar.open('Invalid PrivateKey!', 'Close');
       return;
     }
 
+    if (!validatePrivateStoredWallet(privateWallet)) {
+      this.snackBar.open('Invalid Wallet info!', 'Close');
+      return;
+    }
```

Comment:
`privateWallet ` contains `privateWallet.privateKey`
So, I think it is better to check `privateWallet.privateKey` first